### ISSUE TITLE
export 'IGridsterOptions' was not found in './gridster.service'

### ISSUE THREAD

**Issue by fonzy2013** [swiety85/angular2gridster#9]
I was superhappy seeing that there was a gridster implementation for angular2 but I do get an error when running it on angular-cli that I want to see if you have seen:

```
[fondberg@Niklass-MacBook-Pro teamcity-noc]$ ng serve
** NG Live Development Server is running on http://localhost:4200. **
Hash: 7f1850860d0a5e950f66
Time: 10063ms
chunk    {0} main.bundle.js, main.bundle.map (main) 13.7 kB {2} [initial] [rendered]
chunk    {1} styles.bundle.js, styles.bundle.map (styles) 9.96 kB {3} [initial] [rendered]
chunk    {2} vendor.bundle.js, vendor.bundle.map (vendor) 3.71 MB [initial] [rendered]
chunk    {3} inline.bundle.js, inline.bundle.map (inline) 0 bytes [entry] [rendered]

WARNING in ./~/angular2gridster/src/gridster.component.ts
49:55-71 "export 'IGridsterOptions' was not found in './gridster.service'

WARNING in ./~/angular2gridster/src/gridster.component.ts
49:91-107 "export 'IGridsterOptions' was not found in './gridster.service'

ERROR in Error encountered resolving symbol values statically. Calling function 'makeDecorator', function calls are not supported. Consider replacing the function or lambda with a reference to an exported function, resolving symbol NgModule in /Users/fondberg/Documents/dashboard/teamcity-noc/node_modules/angular2gridster/node_modules/@angular/core/src/metadata/ng_module.d.ts, resolving symbol NgModule in /Users/fondberg/Documents/dashboard/teamcity-noc/node_modules/angular2gridster/node_modules/@angular/core/src/metadata.d.ts, resolving symbol NgModule in /Users/fondberg/Documents/dashboard/teamcity-noc/node_modules/angular2gridster/node_modules/@angular/core/src/core.d.ts, resolving symbol NgModule in /Users/fondberg/Documents/dashboard/teamcity-noc/node_modules/angular2gridster/node_modules/@angular/core/index.d.ts, resolving symbol GridsterModule in /Users/fondberg/Documents/dashboard/teamcity-noc/node_modules/angular2gridster/src/gridster.module.ts, resolving symbol GridsterModule in /Users/fondberg/Documents/dashboard/teamcity-noc/node_modules/angular2gridster/src/gridster.module.ts

ERROR in ./src/app/app.component.ts
Module build failed: Error: /Users/fondberg/Documents/dashboard/teamcity-noc/src/app/app.component.ts (2,55): Module '"/Users/fondberg/Documents/dashboard/teamcity-noc/node_modules/angular2gridster/index"' has no exported member 'GridsterComponent'.)
    at _checkDiagnostics (/Users/fondberg/Documents/dashboard/teamcity-noc/node_modules/@ngtools/webpack/src/loader.js:145:15)
    at /Users/fondberg/Documents/dashboard/teamcity-noc/node_modules/@ngtools/webpack/src/loader.js:172:17
 @ ./src/app/app.module.ts 16:0-47
 @ ./src/main.ts
 @ multi webpack-dev-server/client?http://localhost:4200/ ./src/main.ts
webpack: bundle is now VALID.
```

**Comment by swiety85** [comment]
The issue is that angular compiler has problem with importing interfaces (IGridsterOptions) from file that exports other services (GridsterService). It is fixed by moving interfaces outside of a service file.
The fix is available on version 0.3.1. NPM will be soon updated with this version as well.

**Comment by fonzy2013** [comment]
Great!
